app/components/PlatformAppPageView.vue.ts

```diff
@@ -0,0 +1,87 @@
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+import { PlatformAppsService, EAppPageSlot } from 'services/platform-apps';
+import { Inject } from 'util/injector';
+import electron from 'electron';
+import Utils from 'services/utils';
+import { Subscription } from 'rxjs';
+
+@Component({})
+export default class PlatformAppPageView extends Vue {
+  @Inject() platformAppsService: PlatformAppsService;
+  @Prop() appId: string;
+  @Prop() pageSlot: EAppPageSlot;
+
+  $refs: {
+    appContainer: HTMLDivElement;
+  };
+
+  resizeInterval: number;
+  containerId: number;
+  loadSub: Subscription;
+
+  mounted() {
+    this.mountContainer();
+
+    this.resizeInterval = window.setInterval(() => {
+      this.checkResize();
+    }, 100);
+
+    this.loadSub = this.platformAppsService.appLoad.subscribe(app => {
+      if (this.appId === app.id) {
+        this.unmountContainer();
+        this.mountContainer();
+      }
+    });
+  }
+
+  mountContainer() {
+    this.containerId = this.platformAppsService.mountConatiner(
```

Comment:
There's a typo in this method. Would it make sense to refactor it so it takes an object instead, unclear how the two last args differ?